services/payload/src/components/SideNav/SideNav.scss

```diff
@@ -0,0 +1,78 @@
+.navbar-wrapper {
+    background-color: #FFFFFF;
+    box-shadow: 4px 0px 4px 0px rgba(0, 0, 0, 0.10);
+    display: flex;
+    flex-direction: column;
+    height: 100vh;
+    align-items: center;
+    position: fixed;
+    z-index: 100;
+    width: 240px;
+}
+
+.navbar-icon-wrapper {
+    margin-top: 60px 
+}
+
+.navbar-buttons-wrap {
+    display: flex;
+    flex-direction: column;
+    height: 215px;
```

Comment:
Fixed heights/margins like this are going to lead to really rigid, awkward layouts usually, so I try and avoid them in favor of flexbox/gap if I can!